releasenotes/notes/add-limit-for-dns-stats-be2110d5d18ad671.yaml

```diff
@@ -0,0 +1,4 @@
+fixes:
+  - |
+    Adds a limit to the number of DNS stats objects the DNSStatkeeper can have at any given time.
```

Comment:
```suggestion
    Adds a limit to the number of DNS stats objects the DNSStatkeeper can have at any given time. This can alleviate memory issues on hosts doing high numbers of DNS requests where network performance monitoring is enabled. 
```